plugins/entando-plugin-jpdigital-exchange/src/main/java/org/entando/entando/web/digitalexchange/install/DigitalExchangeInstallResource.java

```diff
@@ -41,14 +41,32 @@
     })
     @RestAccessControl(permission = Permission.SUPERUSER)
     @PostMapping(value = "{exchange}/install/{component}", produces = MediaType.APPLICATION_JSON_VALUE)
-    ResponseEntity<SimpleRestResponse<ComponentInstallationJob>> install(@PathVariable("exchange") String digitalExchangeId,
-            @PathVariable("component") String componentId, HttpServletRequest request) throws URISyntaxException ;
+    ResponseEntity<SimpleRestResponse<DigitalExchangeJob>> install(@PathVariable("exchange") String digitalExchangeId,
+                                                                   @PathVariable("component") String componentId, HttpServletRequest request) throws URISyntaxException ;
+
+    @ApiOperation(value = "Starts component remove job ")
+    @ApiResponses({
+            @ApiResponse( code = 201, message = "Created")
+    })
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @PostMapping(value = "{exchange}/uninstall/{component}", produces = MediaType.APPLICATION_JSON_VALUE)
+    ResponseEntity<SimpleRestResponse<DigitalExchangeJob>> uninstall(@PathVariable("exchange") String digitalExchangeId,
```

Comment:
I think that the uninstall endpoint shouldn't depend by the exchange id. One should be able to remove a component even if the DE that provided it doesn't exist anymore.

Reply:
I've updated the code for this, but before merging we should discuss if the job entry in the database at this point should have nullable DE id and url or the installation job should be saved somewhere else